### PR TITLE
Fix docs for employeeProd and add employeeJobs.

### PR DESCRIPTION
**Pull request by smmalis37**
This is a documentation fix and feature addition. The documentation for the employeeProd field of Office was incorrect, now fixed, and a new employeeJobs field has been added to do what employeeProd previously claimed to do. ~~This has not yet been tested, as I am literally installing npm right now.~~

### CODE REVIEW

**Comment by phyzical** [comment]
imo lets make another func calculateTotalEmployees call it just below the training assign at the top of process

then we can look at reusing this for things like the assign to job ina future pr

**Comment by phyzical** [comment]
Oh and you also need to update the comments in the `bitburner.d.ts` as this is what updates the docs

**Comment by smmalis37** [comment]
This has now been tested:
image
image
image

File: src/NetscriptFunctions/Corporation.ts [hunk 1]
```diff
@@ -753,6 +753,14 @@ export function NetscriptCorporation(
           "Research & Development": office.employeeProd[EmployeePositions.RandD],
           Training: office.employeeProd[EmployeePositions.Training],
         },
+        employeeJobs: {
+          Operations: office.employeeJobs[EmployeePositions.Operations],
+          Engineer: office.employeeJobs[EmployeePositions.Engineer],
+          Business: office.employeeJobs[EmployeePositions.Business],
+          Management: office.employeeJobs[EmployeePositions.Management],
+          "Research & Development": office.employeeJobs[EmployeePositions.RandD],
+          Training: office.employeeJobs[EmployeePositions.Training],
```
Comment (phyzical) [inline src/NetscriptFunctions/Corporation.ts:762]: we still need to expose total

**Review by phyzical — APPROVED** [review]

File: src/NetscriptFunctions/Corporation.ts (hunk 1 above)
Comment (smmalis37) [reply src/NetscriptFunctions/Corporation.ts:762]: Total is not exposed in the EmployeeJobs interface currently, and as such the employeeProd field above doesn't expose it either. Should they both?

**Comment by smmalis37** [comment]
Ping on this? Anything else needed or is it ready to go?

**Comment by hydroflame** [comment]
> smmalis37 commented [1 hour ago]

Uh that's lucky I just looked at it.

So while the added information this PR provides is TECHNICALLY available without it, I'll merge this because it's REALLY convenient.

**Pull request merged**